index.html

```diff
@@ -1,155 +1,236 @@
+        </li>
+        <li>
+          <a href="#" class="iconList">
+            <i class="fa-brands fa-instagram"></i>
+          </a>
+        </li>
+      </ul>
+      <a href="#" class="seeProject" id="resume">Get my resume</a>
+      <ul class="languageAndFramework">
+        <li class="languages">
+          <div class="langItems">
             <p id="languages">languages</p>
             <i class="fa-solid fa-chevron-down"></i>
-        </div>
-    <div class="languagesList">
-         <div id="html" class="iconClass"> <img class="languageIcon" src="html.png"><a href="#"> html</a></div>
-         <div id="css" class="iconClass"> <img class="languageIcon" src="css.png"><a href="#">css</a></div>
-         <div id="js" class="iconClass"> <img class="languageIcon" src="js.png"><a href="#"> javascript</a></div>
-    </div>
-</li>
-
-    <li class="framework fr1"><p id="frameworks">Frameworks</p><i class="fa-solid fa-chevron-right"></i></li>
-    <li class="framework fr2"><p id="skills">Skills</p><i class="fa-solid fa-chevron-right"></i></li>
-  
-</ul>
-</section>
-
-</body>
-</html>
\ No newline at end of file
+          </div>
+          <div class="languagesList">
+            <div id="html" class="iconClass">
+              <img class="languageIcon" src="html.png" />
+              <a href="#">html</a>
+            </div>
+            <div id="css" class="iconClass">
+              <img class="languageIcon" src="css.png" />
+              <a href="#">css</a>
+            </div>
+            <div id="js" class="iconClass">
+              <img class="languageIcon" src="js.png" />
+              <a href="#">
+                javascript
+              </a>
+            </div>
+          </div>
+        </li>
+
+        <li class="framework fr1">
+          <p id="frameworks">Frameworks</p>
+          <i class="fa-solid fa-chevron-right"></i>
+        </li>
+        <li class="framework fr2">
+          <p id="skills">Skills</p>
+          <i class="fa-solid fa-chevron-right"></i>
+        </li>
+      </ul>
+    </section>
+
+    <footer class="about-contact">
+      <h1 class="contact-me">Contact me</h1>
+      <p class="contact-p">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I'd love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/xpznolgk" method="post" class="form">
+        <input maxlength="30" class="form-elements" type="text" name="name" placeholder="Yeremias" required />
+        <input
+          class="form-elements"
+          type="email"
+          name="email"
+          placeholder="natajayanj14@gmail.com"
+          required
+        />
+        <textarea name="message" maxlength="500" class="form-elements" cols="50" rows="10" placeholder="Write your message here"></textarea>
+        <input type="submit" value="Get In Touch" id="btn-1"/>
```

Comment:
Great job @Fahim-Khan-P 🤝

- [ ] Kindly implement all button interactions of `hover`, `pressed`, and  `enable` on the `Get In Touch` button.